### PR TITLE
Replace custom required options check with standard PicoCLI features

### PR DESCRIPTION
**Pull request by fab-10**
## PR description

Replace the custom made `validateRequiredOptions`, that check if a required option is also not empty, with PicoCLI standard features and validation using JSR-380 BeanValidation, because the custom made validation breaks some feature of PicoCLI like https://picocli.info/#_mutually_dependent_options

### Thanks for sending a pull request! Have you done the following?

- [ ] Checked out our [contribution guidelines](https://github.com/hyperledger/besu/blob/main/CONTRIBUTING.md)?
- [ ] Considered documentation and added the `doc-change-required` label to this PR [if updates are required](https://wiki.hyperledger.org/display/BESU/Documentation).
- [ ] Considered the changelog and included an [update if required](https://wiki.hyperledger.org/display/BESU/Changelog).
- [ ] For database changes (e.g. KeyValueSegmentIdentifier) considered compatibility and performed forwards and backwards compatibility tests

### Locally, you can run these tests to catch failures early:

- [ ] unit tests: `./gradlew build`
- [ ] acceptance tests: `./gradlew acceptanceTest`
- [ ] integration tests: `./gradlew integrationTest`
- [ ] reference tests: `./gradlew ethereum:referenceTests:referenceTests`

### CODE REVIEW

File: gradle/versions.gradle [hunk 1]
```diff
@@ -157,6 +157,8 @@ dependencyManagement {
 
     dependency 'org.fusesource.jansi:jansi:2.4.1'
 
+    dependency 'org.hibernate.validator:hibernate-validator:8.0.1.Final'
```
Comment (jflo) [inline gradle/versions.gradle:160]: surprised to see this, can we narrow this dependency down to just jakarta?

**Review by jflo — COMMENTED** [review]
love this! note on the dependency on hibernate-validator, seems overly broad?

File: gradle/versions.gradle (hunk 1 above)
Comment (fab-10) [reply gradle/versions.gradle:160]: jakarta is only the API, we need an implementation and it is provided by `hibernate-validator`

**Review by usmansaleem — APPROVED** [review]
Looks straight forward.

**Pull request merged**